docs/guides/advanced-use-case/cross_cluster_backup.md

```diff
@@ -0,0 +1,582 @@
+
+**Create `staging` cluster and install Stash**
+
+Let's create a cluster named `staging`,
+```bash
+$ kind create cluster --name staging
+Creating cluster "staging" ...
+ ✓ Ensuring node image (kindest/node:v1.23.0) 🖼
+ ✓ Preparing nodes 📦  
+ ✓ Writing configuration 📜 
+ ✓ Starting control-plane 🕹️ 
+ ✓ Installing CNI 🔌 
+ ✓ Installing StorageClass 💾 
+Set kubectl context to "kind-staging"
+You can now use your cluster with:
+
+kubectl cluster-info --context kind-staging
+
+Have a nice day! 👋
+```
+
+To verify the current cluster you are working on,
+```
+$ kubectl config current-context
+kind-staging
+```
+
+if you are currently not in the `staging` cluster, you can switch your cluster by the following command,
+```
+$ kubectl config use-context kind-staging
+Switched to context "kind-staging".
+```
+
+We are going to create a namespace named `demo` in the `staging` cluster,
+```
+$ kubectl create ns demo
+namespace/demo created
+```
+
+>**Note:** YAML files used in this tutorial are stored in [docs/examples/guides/advanced-use-case/cross-cluster-backup](/docs/examples/guides/advanced-use-case/cross-cluster-backup) directory of [stashed/docs](https://github.com/stashed/docs) repository.
+
+
+Install `Stash` in your `staging` cluster following the steps [here](/docs/setup/README.md).
+
+
+**Deploy Deployment:**
+
+We are going to create a new Deployment named `stash-recovered` and a PVC as a storage of the Deployment. We will restore the backed-up data inside it.
+
+Here is the YAML of the PVC and the Deployment,
+
+```yaml
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: demo-pvc
+  namespace: demo
+spec:
+  accessModes:
+  - ReadWriteOnce
+  resources:
+    requests:
+      storage: 1Gi
+---
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: stash-recovered
+  name: stash-recovered
+  namespace: demo
+spec:
+  replicas: 3
+  selector:
+    matchLabels:
+      app: stash-recovered
+  template:
+    metadata:
+      labels:
+        app: stash-recovered
+      name: busybox
+    spec:
+      containers:
+      - args:
+        - sleep
+        - "3600"
+        image: busybox
+        imagePullPolicy: IfNotPresent
+        name: busybox
+        volumeMounts:
+        - mountPath: /source/data
+          name: source-data
+      restartPolicy: Always
+      volumes:
+      - name: source-data
+        persistentVolumeClaim:
+          claimName: demo-pvc
+```
+
+Let's create the Deployment we have shown above.
+
+```bash
+$ kubectl apply -f https://github.com/stashed/docs/raw/{{< param "info.version" >}}/docs/examples/guides/advanced-use-case/cross-cluster-backup/deployment_staging.yaml
+persistentvolumeclaim/demo-pvc created
+deployment.apps/stash-recovered created
+```
+
+### Prepare Backend
+
+We are going to restore our backed-up data from a GCS bucket. We have to create a Secret with the necessary credentials and a Repository CRD to use our backend. 
```

Comment:
We have to re-create the Repository CR and the respective Secret that we have used in the `prod` cluster.